pkgs/development/tools/haskell/tinc/default.nix

```diff
@@ -7,12 +7,12 @@
 }:
 mkDerivation {
   pname = "tinc";
-  version = "20160511";
+  version = "20160924";
   src = fetchFromGitHub {
-    owner = "sol";
+    owner = "robbinch";
```

Comment:
Actually, I'm thinking of maintaining this fork just for Nixpkgs. Reasons:
- Last fix is not really perfect in the sense I did not add guards to support older GHC. (It will fail or generate warning messages for older GHCs).
- It is easier to fix tinc just to work with whatever is in Nixpkgs master. Now I only develop Haskell on tinc/nixpkgs so this affects me directly.
- I intend to merge whatever new features @sol makes upstream. He uses tinc daily too, but he's not using NixOS at the moment.
- Lastly, I intend to add more Nixpkgs-specific feature to tinc, specifcally having the ability to pin-down/freeze the version of nixpkgs in use too. This will allow tinc not just to build with the snapshot of hackage packages, but a snapshot of nixpkgs as well. This is at an experimental stage and will probably abuse the tinc code in a way I don't think @sol will approve of for now ;)

Hope this justifies the reason for maintaining my own fork specifically for nixpkgs! If not, I can just use my private copy.